code/roundtrip-tests/pom.xml

```diff
@@ -25,13 +25,26 @@
         <version>1.3.0-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
+
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.sun.xml.fastinfoset</groupId>
     <artifactId>FastInfosetRoundTripTests</artifactId>
     <packaging>jar</packaging>
     <version>1.3.0-SNAPSHOT</version>
     <name>fastinfoset-roundtrip-tests</name>
-    <url>http://fi.java.net</url>
+
+    <properties>
+        <legal.doc.source>${project.basedir}/../..</legal.doc.source>
```

Comment:
`${maven.multiModuleProjectDirectory}` points to the root folder with pom and can help you to avoid redefining this property in child projects

Reply:
Fixed.